libgstc/libgstc_json.c

```diff
@@ -204,3 +204,42 @@ gstc_json_get_child_char_array (const char *json, const char *parent_name,
   return ret;
 
 }
+
+GstcStatus
+gstc_json_child_string (const char *json, const char *parent_name,
+    const char *data_name, char **out)
+{
+  GstcStatus ret;
+  json_t *root;
+  json_t *parent;
+  json_t *data;
+  char *tmp_string;
```

Comment:
Leave some spaces in between the pointer guards.

Reply:
Actually, leave some spaces in this function.

Reply:
This has been updated in the latest version (lines below this)